backend/src/main/java/com/huellapositiva/domain/repository/ESALRepository.java

```diff
@@ -68,6 +68,7 @@ public String saveAsReviser(ESAL model) {
         }
     }
 
+
```

Comment:
One line was not enough?

As an observation, when you send a PR for review you should first check all the changes you made one by one to ensure you don't include changes like this which increase the amount of changed files to review.

Reply:
You have right!!!!